### ISSUE TITLE
Crosstalk with rpivotTable

### ISSUE THREAD

**Issue by njburgoyne** [thomasp85/shinyFiles#34]
Hi nice package, just Started using this package, so I really hope this isn't a stupid issue.

I am trying to load multiple files, and then render some output in a pivottable (using `rpivotTable`).

When I try with both packages in the same app there appears to be a bug that seems to be caused by js crosstalk between the two (I am not an expert so can't tell you where, but the js traceback seems to imply `shinyFiles` calling an `htmlWidgets` function at some point). 

**Example code**

```r
library(shiny)
library(rpivotTable)
library(shinyFiles)

# Define server logic required to draw a histogram
server <- function(input, output, session){
  #volumes <- getVolumes()
  volumes <- c(`Local (C)`="C:", 
               `Data (M)`="M:")
  shinyFileChoose(input = input, 
                  id = 'file', 
                  roots = volumes, 
                  session = session)
  output$filepaths <- renderPrint({parseFilePaths(volumes, input$file)})
  output$bPivotTable <- renderRpivotTable({
    rpivotTable(data = iris)
  })
}

ui <- fluidPage(
  sidebarLayout(
    sidebarPanel(
        fluidRow(strong("Choose files to upload:")),
        fluidRow(
          shinyFilesButton(id = "file", 
                           label = "Files", 
                           title = "Upload Files",
                           multiple = TRUE)
        )
    ),
    mainPanel(
      verbatimTextOutput('filepaths'), 
      rpivotTableOutput("bPivotTable")  
    )
  )
)

print(shinyApp(ui = ui, server = server))
```
just comment out the `rpivotTableOutput("bPivotTable")` line and rerun to see the difference when clicking on the button. (The directories are missing).

The number of dependents on `htmlWidgets` means that a `shinyFiles` fix might be a more logical solution?

**Comment by timelyportfolio** [comment]
ref https://github.com/smartinsightsfromdata/rpivotTable/issues/52

**Comment by thomasp85** [comment]
This is not so much crosstalk between shinyFiles and rpivotTable as it is problems with some combination of Shiny, htmlWidgets, and rpivotTable when calling Shiny.bindAll() (which shinyFiles does, but it is using the shiny API)...

I'll try to raise the issue with the RStudio folks, but other than that there really isn't anything I can do on the shinyFiles side

**Comment by smartinsightsfromdata** [comment]
@thomasp85 @njburgoyne 

I tried to see if I could see anything untoward.  Unfortunately with RStudio Server on OSX (configured with homebrew) & shiny 0.14 I couldn't make the shinyfiles own demo work: 
- The file button gave error
- The folder button showed a directory, but I couldn't click on anything (i.e. I tried with no effects).

Without a proper working demo of shinyfiles I'm stuck.
